### ISSUE TITLE
Potential Scanning Issue

### ISSUE THREAD

**Issue by GiantForkLift** [omkarmoghe/Pokemap#456]
There may be a potential issue with scanning and it not showing all possible pokemon or it may just be something on my end.  I had my Pokemon GO app open and I noticed that there was a Dratini nearby.  So naturally I switched to Pokemaps and scanned the area surrounding me and the Dratini did not show up.  I switched back to Pokemon go to make sure it hadn't despawned and waited for the sightings to update and it was still there.  I then reopened the app to scan and scanned a much larger area this time and still didn't find it but it still showed up in my sightings.  This also happened a few minutes later with an Aerodactyl.

Could it be on my end or possibly something with the latest update?  It could have also been a complete fluke as well.

**Comment by cartifon** [comment]
Well, were you near a lured Pokestop? Because it makes the pokémons to show up but the Pokemap can`t see the Pokémons near it.

**Comment by GiantForkLift** [comment]
No, I was not near a lured stop at the time. Just out on my usual route. I haven't really had more if a chance to replicate it though

**Comment by Kybr-git** [comment]
Same issue here.  Not near any lured stops, not using incense, made sure no filters are on, etc.  I have a second scanner running (pogom) that shows several pokemon (pidgeys, zubats, nidorans, etc) that this scanner is not showing, even after scanning the area multiple times.  Verified that the pokemon were actually there.

**Comment by s7092910** [comment]
Have any of you guys touched the Pokemon Filter setting? Are you not seeing pokemon that could be filtered out and you forgot that you might have filtered them out. Or are you running with 0 pokemon being filtered?

Also are you making sure you aren't scanning too far away from previous scans?

**Comment by GiantForkLift** [comment]
I'm not filtering any of them. And no, unless a few blocks away from a previous scan is too far. Do the sans need to be touching to be completely effective?

**Comment by GiantForkLift** [comment]
I have found that if you wait a few seconds after the scan has finished and then rescan again, everything is more likely to show up.
